examples/java/src/main/java/org/apache/beam/examples/cookbook/TriggerExample.java

```diff
@@ -75,8 +75,8 @@
  *
  * <p>Before running this example, it will be useful to familiarize yourself with Beam triggers
  * and understand the concept of 'late data',
- * See: <a href="http://beam.apache.org/use/walkthroughs/">
- * http://beam.apache.org/use/walkthroughs/</a>
+ * See: <a href="https://beam.apache.org/get-started/wordcount-example/">
```

Comment:
I think this should point to:
https://beam.apache.org/documentation/programming-guide/#triggers